terminology_bot/bot.py

```diff
@@ -67,20 +38,43 @@ def __init__(self):
         self.term_collection = TermCollection()
         self.cur_term = {}
 
-    def set_keyboard(self):
-        """Sets keyboard according to user's language. Default is English."""
-        self.reply_start_menu_keyboard = ReplyKeyboardMarkup([[options['new_term']], [options['list_term']]],
-                                                             resize_keyboard=True, one_time_keyboard=True)
+    def set_language_and_options(self, lang_code):
+        locale_path = 'data/locale/'
+        language = gettext.translation('bot', locale_path, ['en'])
 
-        self.reply_menu_keyboard = ReplyKeyboardMarkup([[options['pos_tag'], options['description']],
-                                                        [options['synonyms'], options['similars']],
-                                                        [options['image'], options['audio'], options['video']]],
-                                                       resize_keyboard=True)
+        try:
+            language = gettext.translation('bot', locale_path, [lang_code])
+        finally:
```

Comment:
Looks strange for me. Why do you use
```
try:
   ...
finally:
   ...
``` 
block. Which Exception are you waiting for?

Reply:
IOError is raised by gettext when there are no translation files needed.